src/plugins/GoldenRetriever/index.js

```diff
@@ -213,9 +213,10 @@ module.exports = class GoldenRetriever extends Plugin {
       this.IndexedDBStore.delete(fileID)
     })
 
-    this.core.on('core:success', (fileIDs) => {
+    this.core.on('core:complete', ({ successful }) => {
+      const fileIDs = successful.map((file) => file.id)
       this.deleteBlobs(fileIDs).then(() => {
-        this.core.log(`[GoldenRetriever] removed ${fileIDs.length} files that finished uploading`)
+        this.core.log(`RestoreFiles: removed ${successful.length} files that finished uploading`)
```

Comment:
Unsure why this is `RestoreFiles` now vs `GoldenRetriever`, thought we deprecated the first?

Reply:
oops, you're right, this must've been a wrongly resolved merge conflict. good eye, thanks! fixed in 7f921672